miq-dev-appliance-setup/MAC/on_appliance/miq-setup.sh

```diff
@@ -6,7 +6,7 @@ cd $MIQ_DIR/vmdb || exit 1
 
 # Update gems.
 echo "**** Updating gems..."
-bundle update || exit 1
+bundle install --without qpid:metric_fu || exit 1
```

Comment:
So, this --without option is a "remembered" option, so when you specify it the first time, it gets saved to the application/.bundle/config file so you shouldn't need it a second time.